_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
+author: Opusforlife, Poolitzer and Allan Nordhøy
 image: newpipe
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+Hey there! Back admittedly a bit late with this release, but for _very_ good reason. Ready? Sit down, grab the virtual popcorn and some hydration.
 <!-- more -->
 
 ## A new unified player
 
-"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
+"If the new player is the _unified player_, was the old one un-unified?" you ask. The new player is new, and it is unified.
 
-A new player it indeed is. Up until now, NewPipe had three different players bundled in its code: the main, pop-up and background players. All three had separate code and separate playing queues. When you switched from one player to another, in the background, the old player and its queue closed itself, while a new player was created, along with a new queue, which took over then. If that sounds complicated and cumbersome, that's because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start from the beginning instead of carrying on from where you left, and you would lose your (possibly carefully built) queue.
```

Comment:
We know it is the new player already, because it already said so. This is spinning dirt with no movement. "Untill now" makes "had" redundant. The rest isn't bad, but it is too long. Move the story along.

Reply:
> We know it is the new player already, because it already said so.

Look at the expression. It is an emphasis. Repetition is often used that way.

> "Until now" makes "had" redundant.

Sorry? "Until now, Newpipe three different players bundled in its code..."?

> The rest isn't bad, but it is too long.

This paragraph suffers mostly from a lack of technical terms for things. "the player buttons below the thumbnail" and "the ones in the drop down menu inside the player" are both phrases that could have been replaced by single words or terms, if they existed.

Reply:
You can say "NewPiped had three different players bundled in its code", or you can get to the point right away. Making it more technical is not have to save this. I'll have a go at trying to streamline matters.

Reply:
> I'll have a go at trying to streamline matters.

Please don't, @comradekingu. This blog post is done. We've moved on. No need to think about it until the next release.